packages/fast-components-react-base/src/slider/slider.tsx

```diff
@@ -894,6 +908,99 @@ class Slider extends Foundation<SliderHandledProps, SliderUnhandledProps, Slider
         return dragValue;
     };
 
+    /**
+     * Handles touch dragging
+     */
+    private handleUpperThumbTouchStart = (e: React.TouchEvent): void => {
+        this.handleThumbTouchStart(e, SliderThumb.upperThumb);
+    };
+
+    private handleLowerThumbMouseTouchStart = (e: React.TouchEvent): void => {
```

Comment:
Should this have "Mouse" in the touch handler name?

Reply:
yep

Reply:
(actually no it shouldn't, but "yep" I handled it...)